packages/server/lib/browsers/memory/cgroup-v1.ts

```diff
@@ -0,0 +1,47 @@
+import { exec } from 'child_process'
+import util from 'util'
+
+const execAsync = util.promisify(exec)
+
+const getTotalMemoryLimit = async () => {
+  return Number((await execAsync('cat /sys/fs/cgroup/memory/memory.limit_in_bytes', { encoding: 'utf8' })).stdout)
```

Comment:
are these paths the same on all machines? seeing similar paths throughout the file

Reply:
I believe for Linux cgroup v1 they are.